nav2_dwb_controller/dwb_plugins/src/limited_accel_generator.cpp

```diff
@@ -63,8 +66,10 @@ void LimitedAccelGenerator::initialize(const nav2_lifecycle::LifecycleNode::Shar
 
 void LimitedAccelGenerator::checkUseDwaParam(const nav2_lifecycle::LifecycleNode::SharedPtr & nh)
 {
-  bool use_dwa;
-  nh->get_parameter_or("use_dwa", use_dwa, true);
+  bool use_dwa = true;
```

Comment:
The default value is set by the `declare_parameter` statement.

Reply:
Yes, true. Same comment as the other one: this code is specifying a default value, but is reading the parameter value into a local variable (not my code, by the way).